### ISSUE TITLE
retrain teacher net 

### ISSUE THREAD

**Issue by wl082013** [irfanICMLL/structure_knowledge_distillation#42]
Hi, 
could you provide the  information how you trained the teacher  network (PSPNet_combine)?
I want to retrain PSPNet, so really appreciate if  you can provide the link or github where you  use the teacher model.
Thanks a lot

**Comment by irfanICMLL** [comment]
I train the code with OCnet two years ago. https://github.com/openseg-group/OCNet.pytorch 

I suggest you use the mmsegmentation, which is a good open tool for semantic segmentation

https://github.com/open-mmlab/mmsegmentation/

**Comment by wl082013** [comment]
okay thanks a lot
I  have been stuck by the synchronous BN as used in your tpami paper (later version). 
As my gpu environment is under pytorch 1.2, cudo 10.0, I have met some difficulty handling this issue.

Could you provide some help replacing the old synchronous BN with the lastest one compiled in pytorch 1.2 or higher ? I have tried to replace it with some,  but they all fail when training several epochs (nan loss). I guess it may be caused by the learning rate, but even when I changed learning rate, it still did not work sometimes.    PSPNet really caused some problems for me. thanks a lot.
